src/main/java/org/ambraproject/wombat/controller/ArticleMetadata.java

```diff
@@ -43,8 +43,7 @@
 import org.springframework.ui.Model;
 
 import javax.servlet.http.HttpServletRequest;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
https://confluence.plos.org/confluence/display/DEV/Java+Coding+Standards no wildcard imports.

Reply:
Good call.  FYI, you need to add some 9's to intellij to keep it from using wildcard imports: https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports